src/core/exception.d

```diff
@@ -88,8 +90,24 @@ class FinalizeError : Error
 
     override char[] toString()
     {
-        auto other = super.next ? super.next.toString : "unknown";
-        return "An exception was thrown while finalizing an instance of class " ~ info.name ~ " :: "~other;
+        char[] chained_exception;
+
+        if (super.next)
+        {
+            char[16] line_buf;
+            cast(void)snprintf(line_buf.ptr, line_buf.length,
```

Comment:
Well, I'm calling C method and I don't check for errors. I like to be explicit ;-)

I can remove it if it bothers you.